src/main/java/racingcar/controller/Controller.java

```diff
@@ -0,0 +1,74 @@
+package racingcar.controller;
+
+import static racingcar.model.utils.common.Validation.validateCarName;
+import static racingcar.model.utils.common.Validation.validateNumberRange;
+import static racingcar.view.RacingCarUserConsole.inputCarName;
+import static racingcar.view.RacingCarUserConsole.inputRacingTry;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.controller.dto.InputDTO;
+import racingcar.controller.dto.OutputDTO;
+import racingcar.model.domain.Cars;
+import racingcar.model.domain.Racing;
+import racingcar.model.domain.RandomMoveBehavior;
+import racingcar.model.domain.VictoryCars;
+import racingcar.model.utils.common.StringUtils;
+import racingcar.view.RacingCarUserResult;
+
+public class Controller {
+
+    private static final int NUMBER_ZERO = 0;
+
+    public static void run() throws Exception {
+        InputDTO inputDto = startGame();
+        Cars cars = new Cars(inputDto.getSplitUserInput());
+        Racing racing = new Racing(cars, RandomMoveBehavior.getInstance());
+        for (int idx = NUMBER_ZERO; idx < inputDto.getRacingTryCounter(); idx++) {
+            racing.drive();
+            List<OutputDTO> result = convertCar(cars);
```

Comment:
`racing.drive()`의 return값이 List<OutboundDTO>여도 좋을 것 같아요!!

```suggestion
            List<OutputDTO> result = racing.drive();
```

Reply:
감사합니다. 🙇‍♂️
마지막 코멘트까지 신경써주셔서 너무 감사드려요.
알려주신 내용 기반으로 다음 미션때 잘 적용해서 조금 더 OOP 다운, TDD 스러운 코드를 작성해볼게요.